src/mgr/DaemonServer.cc

```diff
@@ -927,6 +927,22 @@ bool DaemonServer::_handle_command(
     return true;
   }
 
+  if (prefix == "injectargs") {
+    vector<string> argsvec;
+    cmd_getval(cct, cmdctx->cmdmap, "injected_args", argsvec);
+
+    if (argsvec.empty()) {
```

Comment:
Looks like we are returning `r = -EINVAL` when empty elsewhere - should keep it the same here

Reply:
Thanks Neha. Sending updated one. Will align how it is handled in mgr.